charts/agent/values.yaml

```diff
@@ -1,20 +1,18 @@
-deployment:
-  # Resource settings
-  deployment:
+observe:
+  token:
+  collectionEndpoint:
+
+deployment-cluster-events:
+  deployment-cluster-events:
+    # Resource settings
     resources:
-      limits:
-        cpu: 250m
-        memory: 256Mi
       requests:
         cpu: 250m
         memory: 256Mi
 daemonset:
   daemonset:
     # Resource settings
     resources:
-      limits:
```

Comment:
Thanks for removing these! It will likely reduce some pain for now. Eventually we can provide a config, but that's for later.